### PR TITLE
Json logs

### PR DESCRIPTION
**Pull request by joshuafleck**
The purpose of this change is to allow rabbit feed to log directly to JSON format so that we can send the rabbit feed logs to kibana and search on them. This will improve our ability to debug missing events.

@danisola @pvdb @calo81 @sparrovv @jaynefox Could you please sign off?

### CODE REVIEW

File: lib/rabbit_feed/consumer.rb [hunk 1]
```diff
@@ -7,7 +7,9 @@ module Consumer
     def run
       ConsumerConnection.instance.consume do |raw_event|
         event = Event.deserialize raw_event
+        RabbitFeed.log.info {{ event: :message_received, metadata: event.metadata }}
         event_routing.handle_event event
+        RabbitFeed.log.info {{ event: :message_processed, metadata: event.metadata }}
```
Comment (sparrovv) [inline lib/rabbit_feed/consumer.rb]: Does logstash handle nested JSON ?

**Comment by danisola** [comment]
Shouldn't we also add logging in `ConsumerConnection#handle_message`?

File: lib/rabbit_feed/consumer.rb (hunk 1 above)
Comment (joshuafleck) [inline lib/rabbit_feed/consumer.rb]: Yes, I spoke with @jaynefox about this, and as long as the first log entry has the nested JSON then logstash will handle it

**Comment by joshuafleck** [comment]
@danisola 

> Shouldn't we also add logging in ConsumerConnection#handle_message?

Good point. Fixed in e76aac9

**Comment by danisola** [comment]
Signed-off by @danisola

**Pull request merged**